### PR TITLE
Extend to add ticks

### PR DESCRIPTION
**Pull request by Knight2601**
added renderTicks

```
      renderTicks: {
        divisions: 6,
        divLength: .5,
        divColor: '#fff',
        divWidth: 2,
        subDivisions: 4, 
        subLength: .3,
        subColor: '#fff',
        subWidth: 0.6
      },
```
subTicks are optional as are ticks if the renderTicks is not present
image

see Index.html for example

### CODE REVIEW

**Comment by Knight2601** [comment]
is it worth me nesting divisions so you can go n deep?
e.g. 
```
renderTicks: {
  divisions: {
        divisionCount: 6,
        divLength: .7,
        divColor: '#fff',
        divWidth: 2,
        divisions: {
           divisionCount: 4,
           divLength: .5,
           divColor: '#e8e8e8',
           divWidth: 1.2,
           divisions: {
              divisionCount: 2,
              divLength: .3,
              divColor: '#ccc',
              divWidth: 0.7
           } 
        }
  }
}
```

File: index.html [hunk 1]
```diff
@@ -207,6 +276,14 @@ <h2>Supported browsers</h2>
 
 <h2>Changes</h2>
 
+<h3 id="v1.3.6">Version 1.3.6 (28.11.2017)</h3>
+<p>
+<ul>
+<li>Added support for scalable staticzone sections</li>
+<li>Added optional Ticks(Major/Minor)</li>
+</ul>
```
Comment (kplindegaard) [inline index.html]: Include that two minor issues #146 and #146 were fixed also.

**Review by kplindegaard — COMMENTED** [review]
I trust you have complete control here. Two questions before we merge and release 1.3.6:
1. I see you added test.html, I guess that was intentional?
2. Should we include fixing #146 and #147 in the version history?

Apart from that, I am OK with this and really looking forward to ticking up the version. I actually also like the fact that you used coffee 1.X and not 2.0, because I don't have a good overview of how far back ES6 support goes wrt. browser versions. Most likely not a big concern but you never know...

**Comment by Knight2601** [comment]
Hi 

1.	I don’t know why I use VS Code’s git extensions, that was excluded from the repo, deleted and pushed ( I was testing the render stack while bug fixing the pointer render issue).
2.	If you fixed them in your gh-pages PR then yes - as this is built on top of that.

Yeah I agree about the coffee issue, I have to deal with polyfills quite often with client systems so wouldn’t want to impose anything like that, but I wouldn’t think that it would be an issue as coffee compiles out to JS anyway, so 90% of the systems that consume this would be using the JSmin version anyhow...

have you pulled and seen this working locally? its a quite nice addition to an awesome gauge library :)

**Comment by RobbieTheWagner** [comment]
Is there going to be a release that includes this?

**Pull request merged**